feature/session/src/main/res/layout/item_special_session.xml

```diff
@@ -25,6 +25,27 @@
         android:paddingStart="@dimen/session_bottom_sheet_left_time_space"
         >
 
+        <androidx.emoji.widget.EmojiTextView
```

Comment:
I think this text is not show emoji.So Please use TextView🙏

Reply:
But it uses the robotomono_medium font @takahirom

Reply:
Ok, I'll use TextView.